ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -1219,7 +1219,7 @@ preop_modify(Slapi_PBlock *pb)
         } else {
             /* Subtrees listed on invocation line */
             result = searchAllSubtrees(config->subtrees, config->exclude_subtrees, attrNames, NULL,
-                                       mod->mod_bvalues, requiredObjectClass, sdn, config->unique_in_all_subtrees);
+                                       mod->mod_bvalues, requiredObjectClass, sdn, sdn, config->unique_in_all_subtrees);
```

Comment:
It may be worth renaming sdn here to clarify what's going on to prevent future confusiong :)

Reply:
Sure, thanks!